olp-cpp-sdk-core/src/client/HRN.cpp

```diff
@@ -51,62 +51,54 @@ std::string HRN::ToString() const {
 }
 
 std::string HRN::ToCatalogHRNString() const {
-  if (this->service != ServiceType::Data) return std::string();
+  if (service != ServiceType::Data) return std::string();
 
   return "hrn:" + partition + ":data:" + region + ":" + account + ":" +
-         this->catalogId;
+         catalogId;
 }
 
 HRN::HRN(const std::string& input) {
   Tokenizer tokenizer(input, ':');
 
   // hrn must start with "hrn:"
-  if (!tokenizer.hasNext()) return;
+  if (!tokenizer.HasNext()) return;
```

Comment:
I would add braces.
Same below.

Reply:
@mykhailo-kuchma  Why you didn't add braces here, but in other cases you did?